<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>editPerson</class>
 <widget class="QWidget" name="editPerson">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>622</width>
    <height>220</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLineEdit" name="lineEdit_nameorg_2">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>110</y>
     <width>291</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>100</y>
     <width>221</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 9pt &quot;Segoe UI&quot;;</string>
   </property>
   <property name="text">
    <string>Enter The Next Name Of Role</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_ok">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>160</y>
     <width>83</width>
     <height>29</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>OpenHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>53</x>
     <y>60</y>
     <width>281</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);
font: 700 9pt &quot;Segoe UI&quot;;</string>
   </property>
   <property name="text">
    <string>Enter The Past Name Of Role</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_nameorg">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>70</y>
     <width>291</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>53</x>
     <y>20</y>
     <width>281</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);
font: 700 9pt &quot;Segoe UI&quot;;</string>
   </property>
   <property name="text">
    <string>Enter The Person's Name</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_nameorg_3">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>30</y>
     <width>291</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
